salt/metalk8s/kubeadm/init/certs/ca.sls

```diff
@@ -63,6 +64,14 @@ Create CA salt signing_policies:
             - keyUsage: "critical digitalSignature, keyEncipherment"
             - extendedKeyUsage: "serverAuth"
             - days_valid: {{ ca.signing_policy.days_valid }}
+          kube_user_account_policy:
```

Comment:
I also created this in my PR but lets merge yours first I will update mine

Reply:
OK